libcxx/test/std/lit.local.cfg

```diff
@@ -0,0 +1,5 @@
+# The tests in this directory need to run Python
+import shlex
+import sys
+
+config.substitutions.append(("%{python}", shlex.quote(sys.executable)))
```

Comment:
Since we do this in both `libcxx` and `std`, we might as well put this in `libcxx/test/configs/cmake-bridge.cfg.in` and get rid of both `lit.local.cfg` files.